base/document-download-frontend-deployment.yaml

```diff
@@ -54,7 +54,24 @@ spec:
               value: '$(NEW_RELIC_MONITOR_MODE)'
             - name: NOTIFY_ENVIRONMENT
               value: '$(ENVIRONMENT)'
-          resources: {}
+          resources:
+            requests:
+              cpu: "50m"
+              memory: "400Mi"
+            limits:
+              cpu: "400m"
+              memory: "800Mi"
+          ports:
+            - containerPort: 7001
```

Comment:
Same value than the service https://github.com/cds-snc/notification-manifests/blob/2772a8b4ea096c770538a7858b45ae01242425c7/base/document-download-frontend-service.yaml#L10-L12